src/components/welcome-page/Welcome.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import LinkedInIcon from '@material-ui/icons/LinkedIn';
+import WelcomeBtn from './WelcomeBtn';
+import './Welcome.scss';
+
+const Welcome: React.FC = () => {
+  const text = 'Linked';
+
+  const handleSignIn = () => {
+    console.log('sign in');
+  };
+  const handleSignUp = () => {
+    console.log('sign up');
+  };
+  return (
+    <div className="main-content">
+      <div className="wrapper">
+        <div className="wrapper__header">
+          <span>{text}</span>
+          <LinkedInIcon className="wrapper__header--icon" />
+        </div>
+        <h4>Welcome to your professional community</h4>
```

Comment:
why h4? it could be h1 or h2 because it's the main text on the page / or secondary depends on point of view